### PR TITLE
Fix many spelling errors

### PR DESCRIPTION
**Pull request by IRDonch**
Fix various spelling errors that I found with codespell (and a few that I noticed myself). In a few cases, fix bad grammar, as well.

I left some errors unfixed in the cases where they were a part of the API (primarily ones involving "splitted"), where they were in generated code, or where I didn't fully understand the meaning of the sentence.

### Motivation and context
Bad spelling hurts comprehension and looks unprofessional. :-)

### How has this been tested?
I ran the `cvat-core` unit tests, but other than that, I'm relying on GitHub CI.

### Checklist

- [x] I submit my changes into the `develop` branch
- ~~[ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file~~ seems too minor to warrant changelog inclusion
- ~~[ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly~~ N/A, although some fixes are to the documentation, as well.
- ~~[ ] I have added tests to cover my changes~~
- ~~[ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: site/content/en/docs/for-users/user-guide/advanced/shortcuts.md [hunk 1]
```diff
@@ -29,13 +29,13 @@ Many UI elements have shortcut hints. Put your pointer to a required element to
 |                            | _Modes_                                                                                                  |
 | `N`                        | Repeat the latest procedure of drawing with the same parameters                                          |
 | `M`                        | Activate or deactivate mode to merging shapes                                                            |
-| `Alt+M`                    | Activate or deactivate mode to spliting shapes                                                           |
+| `Alt+M`                    | Activate or deactivate mode to splitting shapes                                                           |
```
Comment (zhiltsov-max) [inline site/content/en/docs/for-users/user-guide/advanced/shortcuts.md:32]:
```suggestion
| `Alt+M`                    | Activate or deactivate mode to splitting shapes                                                          |
```
Adjusted grid

File: site/content/en/docs/for-users/user-guide/advanced/shortcuts.md [hunk 2]
```diff
@@ -29,13 +29,13 @@ Many UI elements have shortcut hints. Put your pointer to a required element to
...
 | `G`                        | Activate or deactivate mode to grouping shapes                                                           |
 | `Shift+G`                  | Reset group for selected shapes (in group mode)                                                          |
 | `Esc`                      | Cancel any active canvas mode                                                                            |
 |                            | _Image operations_                                                                                       |
 | `Ctrl+R`                   | Change image angle (add 90 degrees)                                                                      |
-| `Ctrl+Shift+R`             | Change image angle (substract 90 degrees)                                                                |
+| `Ctrl+Shift+R`             | Change image angle (subtract 90 degrees)                                                                |
```
Comment (zhiltsov-max) [inline site/content/en/docs/for-users/user-guide/advanced/shortcuts.md:38]:
```suggestion
| `Ctrl+Shift+R`             | Change image angle (subtract 90 degrees)                                                                 |
```

**Review by zhiltsov-max — APPROVED** [review]

**Comment by nmanovic** [comment]
@IRDonch , could you please look at linter errors? Looks like in the past we committed  these files with some problems.

**Review by nmanovic — APPROVED** [review]
LGTM

**Pull request merged**